mod_reforged/hooks/contracts/contract.nut

```diff
@@ -0,0 +1,31 @@
+::mods_hookBaseClass("contracts/contract", function(o)
+{
+	while(!("EmployerID" in o.m)) o = o[o.SuperName]; 	// find the base class
```

Comment:
Just use `o = o[o.SuperName]`. There is no need for a while loop because `hookBaseClass` hooks the children of the class, so the SuperName will always be the baseclass itself.